### DISCUSSION TITLE
parallelism solution

### DISCUSSION THREAD

**Discussion by demonic-dev** [salva-imm/algo-and-bs-practice-rs#14] [Q&A] [answered]
are you planning to implement letter-frequency by threads or actor model ?

**Comment by salva-imm** [comment, marked as answer]
well , i guess i'll go with threads , then I'll try `rayon` : )

**Comment by demonic-dev** [comment]
@salva-imm 
great! so what about Actor model ? isn't it suitable for this problem?

**Comment by salva-imm** [comment]
@demonic-dev I guess it's not, the actor model is suitable for problems that you don't want lock mechanism, or you want to scale it easily .
